### PR TITLE
travis osx

### PR DESCRIPTION
**Pull request by vitalyster**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,26 +3,37 @@ python:
...
+      sudo apt-get purge chromium-browser;
+      sudo apt-get install -y --force-yes build-essential prosody ngircd python-sleekxmpp libswiften-dev libprotobuf-dev protobuf-compiler libpurple-dev libglib2.0-dev libdbus-glib-1-dev liblog4cxx10-dev libpopt-dev libboost-dev libboost-signals-dev libboost-system-dev libboost-thread-dev libboost-locale-dev libboost-filesystem-dev libboost-program-options-dev libboost-regex-dev libboost-date-time-dev libcppunit-dev libcommuni-dev libminiupnpc-dev libnatpmp-dev;
+      sudo killall lua5.1;
+    else
+      brew update;
+      brew install prosody ngircd libswiften protobuf pidgin log4cxx popt boost cppunit;
+    fi
 install: "pip install --user sleekxmpp"
 script:
-  - cmake -DCMAKE_BUILD_TYPE=Debug -DENABLE_TESTS=ON . && make && make extended_test
```
Comment (Fank) [inline .travis.yml]: What is the difference bezween `extended_test` and `test` ?

Comment (vitalyster) [inline .travis.yml]: `test` just run library tests, while `extended_test` spin up prosody/ngircd and run some integration tests

Comment (Fank) [inline .travis.yml]: it should be possible on osx or?

Comment (vitalyster) [inline .travis.yml]: It will, I just disabled it while WIP, it needed to have plain test first

**Pull request merged**